### PR TITLE
Export some missing OPRF pieces. Clean up unit tests.

### PR DESCRIPTION
**Pull request by chris-wood**
cc @armfazh @claucece

### CODE REVIEW

File: oprf/oprf_test.go [hunk 1]
```diff
@@ -36,23 +36,7 @@ func TestClientSetUp(t *testing.T) {
 	}
 }
 
-func TestClientRequest(t *testing.T) {
```
Comment (chris-wood) [inline oprf/oprf_test.go:39]: This was redundant with the test(s) below.

File: oprf/oprf_test.go [hunk 2]
```diff
@@ -66,97 +50,41 @@ func TestServerEvaluation(t *testing.T) {
...
-	if err != nil {
-		t.Fatal("invalid evaluation of server: " + err.Error())
-	}
-
-	if eval == nil {
-		t.Fatal("invalid evaluation of server: no evaluation.")
-	}
-}
-
-func TestClientFinalize(t *testing.T) {
```
Comment (chris-wood) [inline oprf/oprf_test.go:84]: This was redundant with the test(s) below.

**Review by armfazh — APPROVED** [review]

File: oprf/oprf.go [hunk 3]
```diff
@@ -195,8 +193,18 @@ func (s *Server) Evaluate(b BlindToken) (*Evaluation, error) {
 	return &Evaluation{ser}, nil
 }
 
+func mustWrite(h io.Writer, data []byte) {
+	dataLen, err := h.Write(data)
+	if err != nil {
```
Comment (claucece) [inline oprf/oprf.go:198]: Write from Hash will never error, why is this checked then?

https://golang.org/pkg/hash/#Hash

```
    // Write (via the embedded io.Writer interface) adds more data to the running hash.
    // It never returns an error.
    io.Writer
```

Comment (chris-wood) [reply oprf/oprf.go:198]: Defensive programing, in case it does. :)

**Pull request merged**